scenarios/ocd/CreateLinuxVMSecureWebServer/README.md

```diff
@@ -1,170 +1,542 @@
-# Intro to Create a NGINX Webserver Secured via HTTPS
-Welcome to this tutorial where we will create a VM. This tutorial assumes you are logged into Azure CLI already and have selected a subscription to use with the CLI. If you have not done this already. Press b and hit ctl c to exit the program. Following that you can enter 
+# This is a change
 
-'az login' followed by 'az account list --output table' and 'az account set --subscription "name of subscription to use"'
+## Intro to Create a NGINX Webserver Secured via HTTPS
 
+Welcome to this tutorial where we'll guide you through setting up a secure Azure Virtual Machine (VM).
+Before you start:
 
-If you need to install Azure CLI run the following command - curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+1. Make sure you've installed the Azure CLI.
 
+    To install Azure CLI run the following command on your bash terminal:
 
-Assuming the pre requisites are met press space bar to proceed
+    ```bash
+    curl -sL https://aka.ms/InstallAzureCLIDeb | sudo bash
+    ```
 
-## Create a resource Group
-The first thing we need to do is create a resource group. You can do this by running the following command
+2. Ensure you're logged into Azure.
 
-'az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION'
+    ```bash
+    az login
+    ```
+
+3. Verify you've selected the correct subscription.
+
+    ```bash
+    az account list --output table #To see the list of subscriptions available to you
+    az account set --subscription <subscription-name-or-id>
+    ```
+
+Instructions:
+
+- If you've met all the prerequisites, press the space bar to continue.
+- If not, press 'b', then 'Ctrl + C' to exit. Follow the subsequent steps provided. Once completed, you can return to and restart the initial program.
+
+## Variable Declaration
+
+List of all the environment variables you'll need to execute this tutorial:
 
 ```bash
-az group create --name $RESOURCE_GROUP_NAME --location $RESOURCE_LOCATION
+export UNIQUE_POSTFIX="$(($RANDOM % 254 + 1))"
+export MY_RESOURCE_GROUP_NAME="myResourceGroup$UNIQUE_POSTFIX"
+export MY_KEY_VAULT="myKeyVault$UNIQUE_POSTFIX"
+export MY_LOCATION="eastus"
```

Comment:
just a heads up that we've renamed `MY_LOCATION` to `REGION` to facilitate region selection by the user

Reply:
Thanks @rguptar. I have updated it.